lib/openxml/elements/paragraph.rb

```diff
@@ -0,0 +1,8 @@
+module OpenXml
+  module Elements
+    class Paragraph < OpenXml::Container
+      namespace :a
```

Comment:
Yeah, looking through the specs, DrawingML has its own `p` and `r` and `t` elements, and they _do_ differ from their WordprocessingML counterparts. So... full steam ahead!